oauth/authentication-oauth2.0.md

```diff
@@ -63,23 +63,30 @@ POST https://app.solvexia.com/oauth/token
 |token_type|`string`|Will always be "Bearer".|
 |expires_in|`number`|Seconds in which the access token will expire, for the user application to handle.|
 
----
+##### Example
+
+```bash
+curl -X POST "https://app.solvexia.com/oauth/token" \
+     -H "Content-Type: application/json" \
+     -d '{"client_id":"DDF-AAFBD447-55432-475B-83DB-B5AD78878821","client_secret":"B5AD78878821","grant_type":"client_credentials"}'
+```
+
 ## Authorize Code Flow
 
-Authorize Code Flow is for client public applications
+Authorize Code Flow is for client public applications.
 
 The flow contains the following steps:
-1. Go to authorize endpoint with client_id, code and redirect url
-2. Manually login
-3. Authorize application to act on your behalf
-4. Generate access token & refresh token by supplying authorisation code and client id
```

Comment:
The majority of the documentation uses American English ("Authorization", "Authorize"). For consistency, change "Authorisation" to "Authorization"